src/utils/getFilteredPeople.tsx

```diff
@@ -0,0 +1,62 @@
+import { Person } from '../types';
+
+function getCentury(year: number) {
+  return Math.ceil(year / 100).toString();
+}
+
+export const getFilteredPeople = (
+  query: string,
+  sex: string,
+  centuries: string[],
+  people: Person[],
+): Person[] => {
+  let filteredPeople = [...people];
+
+  if (query) {
+    filteredPeople = filteredPeople.filter(person => {
+      if (person.name.toLowerCase().includes(query.toLowerCase())) {
+        return person;
+      }
+
+      if (
+        person.motherName &&
+        person.motherName.toLowerCase().includes(query.toLowerCase())
+      ) {
+        return person;
+      }
+
+      if (
+        person.fatherName &&
+        person.fatherName.toLowerCase().includes(query.toLowerCase())
+      ) {
+        return person;
+      }
+
+      return;
+    });
+  }
+
+  if (sex) {
+    filteredPeople = filteredPeople.filter(person => {
+      if (person.sex === sex) {
+        return person;
+      }
```

Comment:
The `return person;` statement is unnecessary in a filter function. You can simply return `true` to include the person in the filtered results.